src/benchmarks/micro/libraries/System.Runtime/Perf.Boolean.cs

```diff
@@ -12,8 +12,19 @@ namespace System.Tests
     [BenchmarkCategory(Categories.Libraries)]
     public class Perf_Boolean
     {
-        public static IEnumerable<object> StringValues => Values.Select(value => value.ToString()).ToArray();
-
+        public static IEnumerable<object> StringValues => new string[]{
+            "true", // strings, not bools, as otherwise strings in the logs have casing "True"
+            "false"
+        };
+        public static IEnumerable<object> AllStringValues => StringValues.Concat(new string[]
+        {
+            "TrUe",
+            "fAlSe",
+            "falss",
```

Comment:
Do we really care about the perf of some of these? In what situation is someone commonly found to have fALSe, for example? What about the values we exist to be most common, i.e. True, true, False, false, TRUE, FALSE? I expect it'll be exceedingly rare to see anything else (for the success case). 

Also, for the success case, why are we testing different values with Parse and TryParse?